sktime/forecasting/model_selection/_split.py

```diff
@@ -1478,7 +1441,6 @@ def _split_by_fh(
     if X is None:
         return y_train, y_test
 
-    else:
-        X_train = X.loc[train]
-        X_test = X.loc[test]
-        return y_train, y_test, X_train, X_test
+    X_train = X.loc[train]
+    X_test = X.loc[test]
+    return y_train, y_test, X_train, X_test
```

Comment:
why?

Reply:
same as in the other comment (https://github.com/sktime/sktime/pull/4108#discussion_r1070364781)